internal/fusemount/fusefs.go

```diff
@@ -114,7 +114,7 @@ func (dir *fuseDirectoryNode) directory() fs.Directory {
 }
 
 func (dir *fuseDirectoryNode) Lookup(ctx context.Context, fileName string, out *fuse.EntryOut) (*gofusefs.Inode, syscall.Errno) {
-	entries, err := dir.directory().Readdir(ctx)
+	e, err := fs.IterateEntriesAndFindChild(ctx, dir.directory(), fileName)
```

Comment:
can we call `Child()` here ?

Reply:
in general IterateEntriesAndFindChild() should only ever be used as an implementation helper for `Child()`